lib/player.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class Player
+    
+    attr_reader :name
```

Comment:
This can also be on the same line, as you may or may not already know.

```
attr_reader :foo, :bar_chart: 
```
[`OPTIONAL`]

Reply:
fixed